swap/htlc.go

```diff
@@ -258,14 +378,20 @@ func (h *Htlc) AddTimeoutToEstimator(estimator *input.TxWeightEstimator) {
 	// Calculate maximum timeout witness size
 	//
 	// - number_of_witness_elements: 1 byte
-	// - sender_sig_length: 1 byte
+	// - sender_key_length: 1 byte
+	// - sender_key: 33 byte (htlc v2)
+	// - sender_sig_length: 1 bytes (htlc v2)
 	// - sender_sig: 73 bytes
 	// - zero_length: 1 byte
 	// - zero: 1 byte
 	// - witness_script_length: 1 byte
 	// - witness_script: len(script) bytes
 	maxTimeoutWitnessSize := 1 + 1 + 73 + 1 + 1 + 1 + len(h.Script)
 
+	if h.Version == HtlcV2 {
```

Comment:
in HtlcV2, the zero_length would be absent

Reply:
done

Reply:
Cool, potentially a unit test would have caught a discrepancy as well but just remember if we fix the v1 that the difference is -1 byte for the removal of the unnecessary 00 push and +1 byte for the addition of the pubkey push